src/Modules/Isrs.asm

```diff
@@ -148,10 +148,11 @@ t1_int_decode_lsb:
     sjmp t1_int_decode_checksum
 
 t1_int_outside_range:
-    inc  Rcp_Outside_Range_Cnt
+    ; Increment Rcp_Outside_Range_Cnt and clip it to 255
     mov  A, Rcp_Outside_Range_Cnt
-    jnz  ($+4)
-    dec  Rcp_Outside_Range_Cnt
+    add  A, #1
+    subb A, #0                          ; C is 1 on overflow, so substract it to undo overflow
```

Comment:
why not clear C?

Reply:
This is a trick.
In case A overflows in the add instruction, C is set, so I use C to clip to 255, substracting it again, instead of implementing jumps or an if else logic

Reply:
Ah - gotcha. Neat trick.

Reply:
I had to roll back this trick because it changes carry bit and it is no push/pop of PSW in t2_int so there won't be any improvement here

Reply:
Hmm, did you push the rollback though? It seems the trick is still in here...